WSUCptSCapstone-Fall2022Spring2023/psd-gamifiedapp
# UI Integration

Closes #27 

Integrates both of our UI commits into a single scene. This is a minor issue that adds very little new functionality.

Files Changed:

ContinueButtonBehavior.cs: Initiates a UI transition when the continue button is pressed.

UILayoutHandler: Manges UI transitions and UI initializations with optional transition delays.

Tests:

I tested using the following process.

1. Start the Application
2. Confirm test level loaded in level list
3. Click on test level.
4. Confirm description loaded.
5. Click "Continue"
6. Confirm test level loaded and UI changed to Level Player
7. Complete Level
8. Confirm return to level select.

The current build passes this manual test successfully and continuously.
